src/pytest_postgresql/executor_noop.py

```diff
@@ -60,10 +60,14 @@ def version(self):
                     options=self.options
             ) as connection:
                 version = str(connection.server_version)
+                # Pad the version for releases before 10
+                # if not we get 90524 instead of 090524
```

Comment:
Why is the padding needed? the parse_version output can be safely compared I think? OR was there a different issue?

Reply:
When I was running it locally, it would parse the version for the 9.x releases incorrectly. 90524 became 90.52 instead of 9.5